src/controllers/update/update.filing.date.controller.ts

```diff
@@ -45,6 +47,578 @@ export const post = async(req: Request, res: Response, next: NextFunction) => {
       }
       if (appData.update) {
         appData.update[FilingDateKey] = mapFieldsToDataObject(req.body, FilingDateKeys, InputDateKeys);
+        appData.update.review_trusts = [{
```

Comment:
please make sure not to check this in! Or put under the feature flag.

Reply:
Ah yes, I've made a note to remove it before we merge. Thanks!